src/plotloader/Yaxispanning.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
Apply our styles to this file. Rename it similarly to the suggestion for the heatmap legend module. Also, use camelCase for the name.

Also, rewrite the comments as the purpose of this module differs to what’s stated in the preamble. You can keep an attribution of the original author if you think it should be credited.